plugins/nf-tower/src/main/io/seqera/tower/plugin/TowerFusionToken.groovy

```diff
@@ -236,8 +242,17 @@ class TowerFusionToken implements FusionToken {
      * @param req The HttpRequest to send
      * @return The HttpResponse received
      */
-    private <T> HttpResponse<String> safeHttpSend(HttpRequest req, RetryPolicy<T> policy) {
-        return Failsafe.with(policy).get(
+    private <T> HttpResponse<String> httpSend(HttpRequest req) {
+        try {
+            safeApply(req)
```

Comment:
Won't be better to name it `safeHttpSend`?

Reply:
Fair! 295d48b7